### PR TITLE
beginning of components

### PR DESCRIPTION
**Pull request by ghost**
added login, and landingpage beginning

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,28 +1,27 @@
 <template>
   <div id="app">
```
Comment (Arcaster42) [inline src/App.vue:2]: Consider using the BootstrapVue containers. So it'll be like this for easy formatting:
`<b-container>
    <b-row>
        <b-col>
`

File: src/components/LandingPage.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          <h1 class="text-white font-weight-bold">SuperDel</h1>
+          <hr class="divider my-4">
+        </div>
+        <div class="col-lg-8 align-self-baseline">
+          <p class="text-white-75 mb-5">Grocery shopping made easy and accessible for all!</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="social-btns">
```
Comment (Arcaster42) [inline src/components/LandingPage.vue:15]: You can use `<b-button>` for the Vue Bootstrap buttons.

File: src/components/Login.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+<script>
+import LoginUser from './LoginUser'
+
+export default {
+  name: 'Login',
+  components: {
+    LoginUser
+  },
+  props: {
```
Comment (Arcaster42) [inline src/components/Login.vue:24]: Store local things in data. Because we are using Vuex, we won't have a need for props.

File: src/main.js [hunk 4]
```diff
@@ -1,8 +1,27 @@
...
+Vue.use(BootstrapVue);
+
+// add vuex to package.json
+import Vuex from "vuex";
+
+Vue.use(Vuex);
+
+const store = new Vuex.Store({
+  state: {
+    currentview: "landingPage",
```
Comment (Arcaster42) [inline src/main.js:16]: Create a `user: null` in here as well, please. We will keep a persistent user object in here upon successful login.

**Review by Arcaster42 — APPROVED** [review]

File: src/components/LoginUser.vue [hunk 5]
```diff
@@ -0,0 +1,26 @@
+<template>
+        <b-form @submit="onSubmit">
+          <b-form-group>
+            <b-form-input v-model="username" type="text" placeholder="Username" required/>
+          </b-form-group>
+          <b-form-group :invalid-feedback="invalidFeedback">
+            <b-form-input v-model="password" type="password" placeholder="Password" required/>
```
Comment (imCaseSensitive) [inline src/components/LoginUser.vue:7]: If you weren't aware, 'v-model' is awesome with computed properties. You can link them directly, and change your vuex state smoothly without having to worry about methods.

**Pull request merged**